internal/templates/image-based-installer/template.go

```diff
@@ -144,6 +144,11 @@ spec:
   externallyProvisioned: true
 {{ if .SpecialVars.CurrentNode.CPUArchitecture }}
   architecture: "{{ .SpecialVars.CurrentNode.CPUArchitecture }}"
+{{ else if and
+    (.Spec.CPUArchitecture)
+    (ne .Spec.CPUArchitecture "multi")
+}}
+  architecture: "{{ .Spec.CPUArchitecture }}"
```

Comment:
IBI only support SNO!

Reply:
I can remove this, I thought it would be better if the templates were consistent though. My documentation will probably need an idea to clarify this limitation as well

Reply:
Yes - please clean this up to avoid confusion and update the doc accordingly.

Reply:
Done on both counts